### PR TITLE
add query schema validate ，and skip error query schema

### PR DESCRIPTION
**Pull request by hyzx86**
Due to translation problems in earlier versions, the wrong query schema content was filled. But due to this wrong query schema, the entire graphQL available API list cannot be loaded

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Queries/Sql/GraphQL/SqlQueryFieldTypeProvider.cs [hunk 1]
```diff
@@ -42,18 +45,23 @@ public async Task<IChangeToken> BuildAsync(ISchema schema)
...
+                    else
+                    {
+                        schema.Query.AddField(BuildSchemaBasedFieldType(schema, query, querySchema));
+                    }
                 }
-                else
+                catch (Exception e)
                 {
-                    schema.Query.AddField(BuildSchemaBasedFieldType(schema, query, querySchema));
+                    _logger.LogError($"The Query {name}'s Schema parse error, Please check the Schema is a correct json format , Schema:{query.Schema}, {System.Environment.NewLine} Error Details:'{e}");
```
Comment (deanmarcussen) [inline src/OrchardCore.Modules/OrchardCore.Queries/Sql/GraphQL/SqlQueryFieldTypeProvider.cs:64]: This is the incorrect logging technique.
It allocates every time when using `$`, but there is a format below, which does not
`_logger.LogError("The query {Name}", name);`

File: src/OrchardCore.Modules/OrchardCore.Queries/Controllers/AdminController.cs [hunk 2]
```diff
@@ -155,13 +156,17 @@ public async Task<IActionResult> CreatePost(QueriesCreateViewModel model)
             {
                 return NotFound();
             }
-
+            if (!string.IsNullOrWhiteSpace(query.Schema) && !CheckSchema(query.Schema))
+            {
+                _notifier.Error(H["Schema is not a correct json format."]);
```
Comment (deanmarcussen) [inline src/OrchardCore.Modules/OrchardCore.Queries/Controllers/AdminController.cs:161]: I think this should should set a model state error

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.Queries/Controllers/AdminController.cs:161]: Also would suggest using the text `Invalid schema JSON supplied`

File: src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs [hunk 3]
```diff
@@ -100,10 +102,14 @@ public void Configure(string name, OpenIdConnectOptions options)
         private async Task<OpenIdClientSettings> GetClientSettingsAsync()
         {
             var settings = await _clientService.GetSettingsAsync();
-            if ((await _clientService.ValidateSettingsAsync(settings)).Any(result => result != ValidationResult.Success))
-            {
-                _logger.LogWarning("The OpenID Connect module is not correctly configured.");
+            var errorResult = (await _clientService.ValidateSettingsAsync(settings)).Where(r => r != ValidationResult.Success);
 
+            if (errorResult.Any())
```
Comment (deanmarcussen) [inline src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs:107]: Can you remove this change, you have it on another pr

File: src/OrchardCore.Modules/OrchardCore.Queries/Sql/GraphQL/SqlQueryFieldTypeProvider.cs (hunk 1 above)
Comment (hyzx86) [reply src/OrchardCore.Modules/OrchardCore.Queries/Sql/GraphQL/SqlQueryFieldTypeProvider.cs:64]: ah ，Thanks   . This is a new knowledge point for me !

**Comment by hyzx86** [comment]
https://github.com/OrchardCMS/OrchardCore/pull/6536

**Pull request closed without merging**